src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/conditions/ElectronicMonitoringProgrammeServiceTest.kt

```diff
@@ -68,126 +68,95 @@ class ElectronicMonitoringProgrammeServiceTest {
 
-      assertThat(licenceCaptor.value.electronicMonitoringProvider)
-        .extracting("isToBeTaggedForProgramme", "programmeName")
-        .containsExactly(
-          true,
-          "Programme Name",
-        )
-    }
+    assertThat(licence.electronicMonitoringProvider).isNull()
   }
 
-  @Nested
-  inner class `feature toggle electronicMonitoringResponseHandlingEnabled` {
-
-    @Test
-    fun `should handle updated conditions when feature toggle is enabled`() {
-      val licenceEntity = aLicenceEntity
-
-      whenever(
-        policyService.isElectronicMonitoringResponseRequired(
-          conditionCodes,
-          licenceEntity.version!!,
-        ),
-      ).thenReturn(true)
-
-      serviceWithFeatureEnabled.handleUpdatedConditionsIfEnabled(licenceEntity, conditionCodes)
-
-      verify(policyService, times(1)).isElectronicMonitoringResponseRequired(conditionCodes, licenceEntity.version!!)
-    }
-
-    @Test
-    fun `should not handle updated conditions when feature toggle is disabled`() {
-      val licenceEntity = aLicenceEntity
-
-      service.handleUpdatedConditionsIfEnabled(licenceEntity, conditionCodes)
-
-      verify(policyService, times(0)).isElectronicMonitoringResponseRequired(any(), any())
-    }
-
-    @Test
-    fun `should handle removed conditions when feature toggle is enabled`() {
-      val licenceEntity = aLicenceEntity
-
-      whenever(
-        policyService.getConditionsRequiringElectronicMonitoringResponse(
-          licenceEntity.version!!,
-          conditionCodes,
-        ),
-      ).thenReturn(
-        listOf(policyApCondition),
-      )
-      whenever(
-        policyService.isElectronicMonitoringResponseRequired(
-          licenceEntity.additionalConditions.map { it.conditionCode }
-            .toSet(),
-          licenceEntity.version!!,
-        ),
-      ).thenReturn(true)
-
-      serviceWithFeatureEnabled.handleRemovedConditionsIfEnabled(licenceEntity, conditionCodes)
-
-      verify(
-        policyService,
-        times(1),
-      ).isElectronicMonitoringResponseRequired(
-        licenceEntity.additionalConditions.map { it.conditionCode }.toSet(),
-        licenceEntity.version!!,
-      )
-    }
-
-    @Test
-    fun `should not handle removed conditions when feature toggle is disabled`() {
-      val licenceEntity = aLicenceEntity
-
-      whenever(
-        policyService.getConditionsRequiringElectronicMonitoringResponse(
-          licenceEntity.version!!,
-          conditionCodes,
-        ),
-      ).thenReturn(
-        listOf(policyApCondition),
-      )
-      whenever(
-        policyService.isElectronicMonitoringResponseRequired(
-          licenceEntity.additionalConditions.map { it.conditionCode }
-            .toSet(),
-          licenceEntity.version!!,
-        ),
-      ).thenReturn(true)
-
-      service.handleRemovedConditionsIfEnabled(licenceEntity, conditionCodes)
-
-      verify(policyService, times(0)).isElectronicMonitoringResponseRequired(any(), any())
```

Comment:
these tests were a bit naff - only checked to see if we called a service to perform a check rather than test the side effects of calling the function